airbyte-webapp/src/locales/en.json

```diff
@@ -125,14 +124,24 @@
   "connectionForm.destinationUse": "Use existing destination",
   "connectionForm.sourceTitle": "Source",
   "connectionForm.destinationTitle": "Destination",
+  "connectionForm.cursor.info": "Used in incremental sync mode only. It determines which records to sync.",
+  "connectionForm.primaryKey.info": "Used in Deduped and History modes only. It determines the unique identifier.",
+  "connectionForm.source.info": "<b>Stream name</b> is the raw table name from your source.",
+  "connectionForm.syncType.info": "How data will be replicated from source to destination.",
+  "connectionForm.destinationName.info": "<b>Namespace</b> is the database schema of your destination tables.",
+  "connectionForm.destinationStream.info": "<b>Stream name</b> is the final table name in destination.",
+  "connectionForm.defaultSyncMode": "Set a default sync mode for all streams",
+  "connectionForm.defaultSyncMode.message": "Set how Airbyte reads from the source and writes to the destination, across all streams <b>when the option is available</b>.",
+  "connectionForm.bulkEdit.cancel": "Cancel",
+  "connectionForm.bulkEdit.apply": "Apply",
 
   "connectorForm.authenticate": "Authenticate your {connector} account",
   "connectorForm.signInWithGoogle": "Sign in with Google",
   "connectorForm.authenticate.succeeded": "Authentication succeeded!",
   "connectorForm.reauthenticate": "Re-authenticate",
 
-  "form.rawData": "Raw data - no normalization",
-  "form.basicNormalization": "Basic normalization - ",
+  "form.rawData": "Serialized JSON",
+  "form.basicNormalization": "Original structure",
```

Comment:
I believe I may indeed use the wrong names for the wrong values. So `Serialized JSON` is for `basicNormalization` right?

Reply:
That was my understanding. @misteryeo could you maybe have a look at that. And also would like to hear your thoughts around "Serialized JSON" vs something like "Flattened JSON" or "Normalized JSON"?

Reply:
Yeah, I think the name and values are switched here.

I don't have an opinion on the use of "Serialized JSON" - I've not seen or heard any confusion around this naming previously though either on blogs mentioning Airbyte or in our Slack community. Conversely, I've not seen anyone refer to flatted or normalized JSON. @ChristopheDuong since you've been working on normalization, do you have thoughts on this?

Reply:
Switched values

Reply:
"serialized json" sounds like the string representation of a json object, so it would be the raw data instead (the way it was first before switched)?

So this labelling does sound confusing...

Reply:
@ChristopheDuong what do you recommend for labelling? Should we just keep the old version?

Reply:
What was the original labeling?

Reply:
maybe normalized records should not denote anything about "json" (which is the internal representation of records/messages in airbyte protocol). 

Either we keep the old naming or a variant (plus if we are changing the naming completely here, we may have to update a lot of docs too?)

What about `"Normalized Tabular data"`?

and for the raw data, you can name "Serialized JSON blob", or "json blob", "json format"?

Reply:
The originals were "Raw data - no normalization" and "Basic normalization".

The raw one does store data as raw JSON. Given that the new view labels it as normalization options, it may make sense to have "Raw data (JSON)" and "Split into separate tables" or "Normalized tabular data" like Chris suggested.